src/main/java/io/jenkins/plugins/sprp/models/Agent.java

```diff
@@ -93,4 +96,12 @@ public void setAnyOrNone(String anyOrNone) {
     public String getAnyOrNone() {
         return anyOrNone;
     }
+
+    public void setTools(HashMap<String, String> tools) {
+        this.tools = tools;
+    }
+
+    public HashMap<String, String> getTools() {
```

Comment:
I do not see an example for that though looks reasonable

Reply:
An example in the readme?

Reply:
Yes